### PR TITLE
Implemented agents

### LINKED ISSUE #24: Implement Agents
https://developers.freshdesk.com/api/#agents

### PR DESCRIPTION
**Pull request by Velua**
This is a follow up of @davinthesmith 's pull request, credits to him!

----

fix #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +182,7 @@ Not implemented: http://developers.freshdesk.com/api/#groups
 - **getCompany(id, callback)** - Get company by ID; [read more](http://developers.freshdesk.com/api/#view_company)
 - **listAllCompanies(callback)** - List all companies; [parameters](http://developers.freshdesk.com/api/#list_all_companies)
 - **updateCompany(id, data, callback)** - Update a company by ID; [parameters](http://developers.freshdesk.com/api/#update_company)
-- **deleteCompany(id, callback)** - Delelete a company by ID, [read more](http://developers.freshdesk.com/api/#delete_company)
```
Comment (maxkoryukov) [inline README.md]: :laughing:

**Comment by maxkoryukov** [comment]
@Velua , need to fix:

- [ ] js doc for new methods
- [x] authors in readme is not updated
- [ ] [currentAgent](https://developers.freshdesk.com/api/#me) still not implemented.

plz, see how the JS-docs are written in the master branch of this repo: https://github.com/arjunkomath/node-freshdesk-api/blob/5baeef3640fd1cb31d41efa09bf10063f77da373/lib/client.js#L301

**Comment by Velua** [comment]
Added the JSDocs for the agents functions, currentAgent function now in there. Take a peek @maxkoryukov

**Comment by maxkoryukov** [comment]
@Velua , why have you added ` .DS_Store` ??? Be carefull;)

**Pull request merged**